EmaPajic/assignment1/assignment1_Q1.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+import sys
+
+def areAnagrams(str1,str2,isCaseSensitive):
+    
+    letters = [0]*257
+    for i in range(0, len(str1)):
+        if ord(str1[i]) <= 90 and ord(str1[i]) >= 65 and isCaseSensitive == 0 :
```

Comment:
I think you can use isalpha() to check if a character is a letter: https://docs.python.org/2/library/stdtypes.html#str.isalpha

Reply:
updated to isupper()